packages/components/README.md

```diff
@@ -110,3 +110,37 @@ As with the browser tests, it is vital that the component to test is mounted and
+would be automatically reflected in Storybook and the Documentation site after a build step in the components package.
+
+Supported statuses are `alpha`, `beta` and `stable`.
+
+```json
+// package.json
+"pieMetadata": {
+  "componentStatus": "alpha"
+},
+```
+
+### Release Categorisation
+
+|                    | Prerelease/Alpha (v0.x.x)                                                                                                           | Beta (v0.x.x)                                                                                                            | Stable (v1.x.x)                                                                                                                                                                                                   |
+|--------------------|-------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **NPM Versioning** | Breaking changes should be expected. Component is still in active development and released as v0.x.x (indicating semver prerelease) | Breaking changes will be less likely, but could still occur due to component being integration tested with applications. | Component moves to v1.x.x NPM release. Breaking changes are communicated through version updates and any major feature changes will be tested as beta package releases.                                           | 
+| **Use in production applications is** | Not recommended. Component is still in active development and may have known issues. Is done so at application teams own risk.      | Is encouraged, but with appropriate level of caution for a beta package release. Thorough testing is recommended.        | Is encouraged and we don’t expect any obvious defects being found.  Component has already been tested in other applications and is therefore considered stable. As always, thorough testing is still recommended. | 
+
+
+
+### What to do as a contributor
+Versioning will happen automatically through our pipeline, and will use each commit description to determine the right version. Remember you can use `yarn cz` when committing any changes. You don't need to change versions manually in the `package.json`.
+
+**When to change a status in `package.json`**
+
+Use `alpha` during development of the firs epic or MVP of the component.
+
+Once the MVP is done, the component will be released as `beta`. This is generally done by the pie team.
+
+It will move to `stable` after it has been tested in at least 2 production environments. This is generally done by the pie team.
```

Comment:
PIE, or Design System (Core)?
```suggestion
Once the MVP is done, the component will be released as `beta`. This is generally done by the PIE team.

It will move to `stable` after it has been tested in at least 2 production environments. This is generally done by the PIE team.
```